syntax = "proto3";

option csharp_namespace = "AcademyCloud.ResourceManagement.Protos.Volume";

package volume;

service VolumeService {
  rpc GetVolumes(GetVolumesRequest) returns (GetVolumesResponse);
}

message GetVolumesRequest {}

message Volume {
  string id = 1;
  int32 size = 2;
  string createTime = 3;
  string attachedToInstanceId = 4;
  string attachedToInstanceName = 5;
  string attachedToDevice = 6;
}

message GetVolumesResponse { repeated Volume volumes = 1; }
